index.js

```diff
@@ -13,9 +13,10 @@ dockerHubAPI.login(process.env.DOCKERHUB_USERNAME, process.env.DOCKERHUB_PASSWOR
     // Load README
     var filePath = process.env.README_PATH || path.join('/data' , 'README.md');
     const readme = fs.readFileSync(filePath, {encoding: 'utf-8'});
+    const desc   = process.env.SHORT_DESCRIPTION || '';
```

Comment:
This should not default to an empty string. Otherwise we reset the currently set short description for all repositories that use this project.

Reply:
I've tested it without specified `-e SHORT_DESCRIPTION` and with `-e SHORT_DESCRIPTION=''`. Both variants are not removing existing short description.